vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthEmailEntryFragment.kt

```diff
@@ -51,7 +60,10 @@ class FtueAuthEmailEntryFragment :
     }
 
     private fun setupViews() {
-        views.emailEntryInput.associateContentStateWith(button = views.emailEntrySubmit, enabledPredicate = { it.isEmail() })
+        views.emailEntryInput.hint = getString(if (params.mandatory) R.string.ftue_auth_email_entry_title else R.string.login_set_email_optional_hint)
+        views.emailEntryInput.associateContentStateWith(button = views.emailEntrySubmit, enabledPredicate = {
```

Comment:
This would read a bit better like this
```
    views.emailEntryInput.associateContentStateWith(
        button = views.emailEntrySubmit,
        enabledPredicate = { it.isEmail() || it.isEmptyAndOptional() }
    )
```

Reply:
agreed, happy to update! my original reasoning was to follow the last argument function convention but with a property name